wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -1,178 +1,92 @@
+| Stapeln | Stapelt die übereinander liegenden Hit-Circles wie im Spiel.[^stacking-note] |
 
 ## Komponieren
 
 *Hauptseite: [Komposition](/wiki/Client/Beatmap_editor/Compose)*
 
 ![Kompositionsmenü](img/M_Compose-DE.jpg "Kompositionsmenü")
 
-### Regler
-
 | Name | Beschreibung |
 | :-- | :-- |
-| Taktteiler | [Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) |
+| Taktteiler | Passe die Feinheit des [Taktteilers](/wiki/Client/Beatmap_editor/Beat_snap_divisor) an, wobei mehr oder weniger Striche auf der [Zeitleiste](/wiki/Client/Beatmap_editor/Timelines) angezeigt werden. |
 | Abspieltempo | Gibt an, wie schnell das Lied abgespielt wird. |
-| Gitter-Ausrichtung (`T`) | Erlaubt das Einrasten der Hit-Objekte in Bezug auf die jeweilige Gitterlinie. |
-
-### Spezielle Befehle
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Polygon-Noten erstellen ... (`Strg` + `Shift` + `D`) | Erstellt einen kreisförmigen Fluss aus Hit-Circles. Du kannst den Distance-Snap (0.1x - 2.0x) {Abstand zwischen den Objekten}, Offsetwinkel (0° - 180°) {Rotationswinkel}, Wiederholungen und Punkte (3 - 14/282) {Anzahl an Objekten pro Schleife} anpassen. |
-| Slider zu Stream konvertieren... | (Wähle einen Slider) Ändere den Slider zu einem Stream aus Hit-Circles. Du kannst zwischen zwei Optionen wählen: nach Objektanzahl oder nach Distance-Snap. **Nach Objektanzahl:** Passe den Slider-Fluss an die gegebene Anzahl der Hit-Circles an (1 - 17/100). **Nach Distance-Snap:** Passe den Slider-Fluss mit einem festgelegtem Abstand zwischen zwei Noten an(0.10x - 5.00x). Dies kann zu einer absurden Anzahl an Hit-Circles führen, da es auf dem Taktteiler beruht (ein Beat = eine Note). **Taktteiler:** Timing-Einsteller für Noten. |
-| Live-Mapping-Modus aktivieren (`Strg` + `Tab`) | osu!/osu!catch/osu!taiko Beatmaps: Benutze die osu!taiko-Tasten, um Hit-Circles, Fruits oder osu!taiko-Hit-Objekte zu platzieren, während du den Song im Editor spielst. Die Objekte werden an der Position des Cursors platziert. osu!mania Beatmaps: Benutze die osu!mania-Tasten, um Hit-Objekte während dem Spielen im Editor zu platzieren. Wird anhand der Taste, die du gedrückt hast, platziert. Funktioniert nicht für Slider/Holds. |
-| Sample importieren | [Tastentöne und Sound-Sampling.](/wiki/Client/Beatmap_editor/Compose) |
+| Gitter-Ausrichtung (`T`) | Erlaubt das [Einrasten](/wiki/Beatmapping/Grid_snapping) der Hit-Objekte in Bezug auf die jeweilige Gitterlinie des Spielfelds. |
+| Polygon-Noten erstellen ... (`Strg` + `Shift` + `D`) | Erstellt anhand des aktiven Taktteilers einen kreisförmigen Fluss aus Hit-Circles. |
+| Slider zu Stream konvertieren ... | Ändere den [Körper](/wiki/Gameplay/Hit_object/Slider/Sliderbody) eines ausgewählten Sliders zu einem Stream aus Hit-Circles. **Nach Objektanzahl:** Verwendung einer bestimmten Anzahl von Hit-Circles. **Nach Abstandsraster:** Platziere Objekte nach einem [festgelegtem Abstand](/wiki/Client/Beatmap_editor/Distance_snap). |
```

Comment:
```suggestion
| Slider zu Stream konvertieren ... | Ändere den [Körper](/wiki/Gameplay/Hit_object/Slider/Sliderbody) eines ausgewählten Sliders zu einem Stream aus Hit-Circles. **Nach Objektanzahl:** Verwendung einer bestimmten Anzahl von Hit-Circles. **Nach Distance Snap:** Platziere Objekte nach einem [festgelegtem Abstand](/wiki/Client/Beatmap_editor/Distance_snap). |
```

I will use `Distance Snap` until we decide to use something else.

As a more native sounding term, I could imagine:

- Distanzraster
- Abstandsraster

Reply:
Vllt Objektdistanz oder Objektabstand für Distance Snap